test/integration/imagechange_buildtrigger_test.go

```diff
@@ -64,6 +69,9 @@ func TestSimpleImageChangeBuildTrigger(t *testing.T) {
 		t.Fatalf("Expected build with base image %s, got %s", "registry:8080/openshift/test-image:ref-2", newBuild.Parameters.Strategy.DockerStrategy.BaseImage)
 	}
 
+	event = <-watch2.ResultChan()
+	event = <-watch2.ResultChan()
```

Comment:
you can see the image_change_controller only updates it once, here:
https://github.com/openshift/origin/blob/master/pkg/build/controller/image_change_controller.go#L81
https://github.com/openshift/origin/blob/master/pkg/build/controller/factory/factory.go#L262